src/channel.ts

```diff
@@ -214,6 +214,8 @@ export function getSupportedMark(channel: Channel): SupportedMark {
       return {point: true, geoshape: true};
     case TEXT:
       return {text: true};
+    default:
```

Comment:
I wonder if this is the right place to correct this?  

If the channel you have isn't a valid `Channel` at all, then it doesn't satisfy the signature of this method. 

A better check would be using `isChannel` method to check if a key of the encoding object  is a channel.

Reply:
Made the changes. Thanks for the help!